WooCommerce/StoreWidgets/Homescreen/StoreInfoView.swift

```diff
@@ -278,12 +278,14 @@ private extension UnableToFetchView {
 }
 
 // MARK: - Previews
+#if DEBUG
```

Comment:
Added this in all of the previews for widget so that the `import class WooFoundation.CurrencySettings` is only used for debug builds